File: 백준/Bronze/1934. 최소공배수/최소공배수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    private static int gcd(int a, int b) {
        return b == 0 ? a : gcd(b, a % b);
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(br.readLine());

        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < t; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            int a = Integer.parseInt(st.nextToken());
            int b = Integer.parseInt(st.nextToken());

            long lcm = (long) a * b / gcd(a, b);
            sb.append(lcm).append('\n');
        }

        System.out.print(sb);
    }
}

File: 백준/Bronze/1934. 최소공배수/최소공배수.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.StringTokenizer

private fun gcd(a: Int, b: Int): Int = if (b == 0) a else gcd(b, a % b)

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val t = br.readLine().toInt()

    val sb = StringBuilder()
    repeat(t) {
        val st = StringTokenizer(br.readLine())
        val a = st.nextToken().toInt()
        val b = st.nextToken().toInt()

        val lcm = a.toLong() * b / gcd(a, b)
        sb.append(lcm).append('\n')
    }

    print(sb)
}
